fastlane_core/lib/fastlane_core.rb

```diff
@@ -1,5 +1,7 @@
 require 'json'
 
+# ENV[] helper
+require 'fastlane_core/env'
```

Comment:
Is there a reason why this is above the monkey-patches? The comment on the next line says that that needs to come first.